superset/assets/src/explore/components/ExploreViewContainer.jsx

```diff
@@ -131,8 +131,6 @@ class ExploreViewContainer extends React.Component {
 
   /* eslint no-unused-vars: 0 */
   componentDidUpdate(prevProps, prevState) {
-    this.triggerQueryIfNeeded();
```

Comment:
This can be removed because the child component is handling the refresh, correct?

Reply:
yes. this line is added from #7233: 
https://github.com/apache/incubator-superset/blob/df9efa8ed9f6ee263f52b791086b286761e9aab6/superset/assets/src/chart/Chart.jsx#L84